tests/github/w3c/IndexedDB/index.html

```diff
@@ -3360,7 +3360,7 @@ <h3 class="heading settled" data-level="4.7" id="keyrange"><span class="secno">4
                                        <c- b>optional</c-> <a class="idl-code" data-link-type="interface" href="https://heycam.github.io/webidl/#idl-boolean" id="ref-for-idl-boolean①⓪"><c- b>boolean</c-></a> <dfn class="idl-code" data-dfn-for="IDBKeyRange/bound(lower, upper, lowerOpen, upperOpen), IDBKeyRange/bound(lower, upper, lowerOpen), IDBKeyRange/bound(lower, upper)" data-dfn-type="argument" data-export id="dom-idbkeyrange-bound-lower-upper-loweropen-upperopen-loweropen"><code><c- g>lowerOpen</c-></code><a class="self-link" href="#dom-idbkeyrange-bound-lower-upper-loweropen-upperopen-loweropen"></a></dfn> = <c- b>false</c->,
                                        <c- b>optional</c-> <a class="idl-code" data-link-type="interface" href="https://heycam.github.io/webidl/#idl-boolean" id="ref-for-idl-boolean①①"><c- b>boolean</c-></a> <dfn class="idl-code" data-dfn-for="IDBKeyRange/bound(lower, upper, lowerOpen, upperOpen), IDBKeyRange/bound(lower, upper, lowerOpen), IDBKeyRange/bound(lower, upper)" data-dfn-type="argument" data-export id="dom-idbkeyrange-bound-lower-upper-loweropen-upperopen-upperopen"><code><c- g>upperOpen</c-></code><a class="self-link" href="#dom-idbkeyrange-bound-lower-upper-loweropen-upperopen-upperopen"></a></dfn> = <c- b>false</c->);
 
-  <a class="idl-code" data-link-type="interface" href="https://heycam.github.io/webidl/#idl-boolean" id="ref-for-idl-boolean①②"><c- b>boolean</c-></a> <a class="idl-code" data-link-type="method" href="#dom-idbkeyrange-_includes" id="ref-for-dom-idbkeyrange-_includes"><c- g>_includes</c-></a>(<c- b>any</c-> <dfn class="idl-code" data-dfn-for="IDBKeyRange/_includes(key)" data-dfn-type="argument" data-export id="dom-idbkeyrange-_includes-key-key"><code><c- g>key</c-></code><a class="self-link" href="#dom-idbkeyrange-_includes-key-key"></a></dfn>);
+  <a class="idl-code" data-link-type="interface" href="https://heycam.github.io/webidl/#idl-boolean" id="ref-for-idl-boolean①②"><c- b>boolean</c-></a> <a class="idl-code" data-link-type="method" href="#dom-idbkeyrange-_includes" id="ref-for-dom-idbkeyrange-_includes"><c- g>_includes</c-></a>(<c- b>any</c-> <dfn class="idl-code" data-dfn-for="IDBKeyRange/includes(key)" data-dfn-type="argument" data-export id="dom-idbkeyrange-includes-key-key"><code><c- g>key</c-></code><a class="self-link" href="#dom-idbkeyrange-includes-key-key"></a></dfn>);
```

Comment:
The change removes the underscore in `data-dfn-for="IDBKeyRange/_includes(key)"`, `id="dom-idbkeyrange-_includes-key-key"` and `href="#dom-idbkeyrange-_includes-key-key"`. That by itself seems harmless.